deals/deals_test.go

```diff
@@ -42,10 +43,25 @@ func TestStore(t *testing.T) {
 	for _, nm := range numMiners {
 		t.Run(fmt.Sprintf("CantMiners%d", nm), func(t *testing.T) {
 			client, _, _ := tests.CreateLocalDevnet(t, nm)
-			m, err := New(client, WithImportPath(filepath.Join(tmpDir, "imports")))
+			m, err := New(tests.NewTxMapDatastore(), client, WithImportPath(filepath.Join(tmpDir, "imports")))
 			checkErr(t, err)
-			_, err = storeMultiMiner(m, client, nm, randomBytes(600))
+			_, pcids, err := storeMultiMiner(m, client, nm, randomBytes(600))
 			checkErr(t, err)
+			pending, err := m.PendingDealRecords()
+			require.Nil(t, err)
+			require.Len(t, pending, nm)
```

Comment:
Inserted checks for the deal records into the storage test. Required refactoring `storeMultiMiner` and `waitForDealComplete` to be separate calls so we can check things in between